t5/convert.py

```diff
@@ -64,5 +66,12 @@ def convert(model_name):
         help="Name of the T5 model.",
         default="t5-small",
     )
+    parser.add_argument(
+        "--dtype",
+        help="The model data type.",
+        type=str,
+        choices=["float16", "float32"],
+        default="float16",
```

Comment:
It's unfortunate that we can't save `bfloat16` that's something that will work once we have safetensor support :). If you think saving to `fp16` is a bad idea, feel free to change the default, I'm not wedded to `fp16` as the default.

Reply:
Oh right, I meant to change it, but forgot. Thanks!

Serializing to `bfloat16` with safetensors will be great! I think most of these models were trained on bfloat16, and the serialization as `float16` is quite destructive.